frappe/__init__.py

```diff
@@ -1985,6 +1985,7 @@ def get_print(
 	doc=None,
 	output=None,
 	no_letterhead=0,
+	letterhead=None,
 	password=None,
 	pdf_options=None,
```

Comment:
can we move the param after `pdf_options`? - might break some code which specifies params by position